.circleci/config.yml

```
@@ -376,6 +376,49 @@ jobs:
               docker push fundocker/mork:latest
             fi
 
+  # Publisk the Helm chart to DockerHub
+  publish-helm:
+    docker:
+      - image: cimg/base:current
+        auth:
+          username: $DOCKER_HUB_USER
+          password: $DOCKER_HUB_PASSWORD
+    working_directory: ~/fun
+    steps:
+      # Checkout repository sources
+      - checkout
+      - *docker-login
+      - run:
+          name: Install helm
+          command: |
+            export HELM_RELEASE="v3.16.2"
```

Maybe remove this `export` and use the Circle CI environment attribute _([doc](https://circleci.com/docs/env-vars/#order-of-precedence))_.

```suggestion
          environment:
            HELM_RELEASE: v3.16.2
          command: |
```

---

Nice, thanks!